packages/apps/graph-client/src/utils/hooks/use-parsed-blocks.ts

```diff
@@ -1,6 +1,9 @@
 import type { GetBlocksSubscription } from '../../__generated__/sdk';
 
 import { useCallback, useState } from 'react';
+import { env } from '../env';
+
+const MAX_HEIGHTS = 7;
```

Comment:
Variable is not used

Reply:
@alber70g I see you added some commits, can you check what these changes do?